1.9.py

```diff
@@ -0,0 +1,14 @@
+#Exercise 2.1
+answer1 = 42*60+42
+print(f'There are {answer1} seconds in 42 minutes 42 seconds\n')
+
+#Exercise 2.2
+answer2  = 10/1.61
+print (f'There are {round(answer2,2)} miles in 10 kilometers\n')
```

Comment:
Why round?

Reply:
Done! thank you. I thought it would look better